proposed/phpdoc.md

```diff
@@ -266,10 +266,8 @@ A Summary MUST contain an abstract of the "Structural Element" defining the
 purpose. It is RECOMMENDED for Summaries to span a single line or at most two,
 but not more than that.
 
-A Summary MUST end with either
-
-* a full stop (.) followed by a line break
-* or two sequential line breaks.
+A Summary MUST end with a full stop (.) and if there is additional content it
+MUST be separated from the Summary by two sequential line breaks.
```

Comment:
I had to read this sentence twice to understand it.  It sounds like "A summary must... and it must be separated from the summary..."  Which makes me think you're separating the summary from the summary. :smile: (I think it's because Summary is the only proper noun in the sentence.)

Suggested alternative: 

```
A Summary MUST end with a full stop (.).  

If there is any additional content, the Summary MUST be followed by two sequential line breaks to separate it from the additional content.
```